api/src/routes/flashcard.rs

```diff
@@ -88,3 +88,47 @@ pub async fn get_flashcard(
 
     HttpResponse::Ok().json(flashcards)
 }
+
+exportable! {
+    pub struct FlashcardFromIDGet {
+        flashcard_id: i32
+    }
+}
+
+exportable! {
+    pub struct FlashcardFromIDResp {
+        id: i32,
+        folder_id: i32,
+        term: String,
+        definition: String
+    }
+}
+
+#[get("/flashcard/fromid")]
+pub async fn get_flashcard_from_id(
+    data: Data<AppState>,
+    info: web::Query<FlashcardFromIDGet>,
+    req: HttpRequest,
+) -> impl Responder {
+    let user_id: i32 = utils::get_user_id(&req).unwrap();
+    let flashcards = sqlx::query_as!(
+        // TODO: replace "select id where id"
```

Comment:
Is this for this PR? What does this mean?

Reply:
Using the `query_as` utility meant one of the columns I was selecting was also one I was providing, which was convenient but strange - actually, I don't need the `id` column any more though, so I've removed this.